apache2.yaml

```diff
@@ -129,6 +129,29 @@ subpackages:
           mv "${{targets.destdir}}"/usr/share/apache2/icons "${{targets.subpkgdir}}"/usr/share/apache2/
           mv "${{targets.destdir}}"/usr/share/apache2/error "${{targets.subpkgdir}}"/usr/share/apache2/
 
+  - name: ${{package.name}}-oci-entrypoint
+    description: "OCI entrypoint for apache2"
+    dependencies:
+      runtime:
+        - ${{package.name}}
+        - dash-binsh
+    pipeline:
+      - runs: |
+          set -x
+          # Add the entrypoint
+          mkdir -p ${{targets.subpkgdir}}/usr/bin
+          mv entrypoint.sh ${{targets.subpkgdir}}/usr/bin/entrypoint.sh
```

Comment:
weird to move this. you could copy or `install`. wierd is just strange because i think the file came from the source.